### PR TITLE
Np 10940/model ipgeolocationinfo

### PR DESCRIPTION
**Pull request by noodlemoodle**

### CODE REVIEW

**Comment by jlhughes** [comment]
Any reason not to place GeolocationSupport under net/ip?  It's all related.

**Comment by noodlemoodle** [comment]
no, i can move it 

> Any reason not to place GeolocationSupport under net/ip? It's all related.

File: src/foam/net/ip/services.jrl [hunk 1]
```diff
@@ -0,0 +1,15 @@
+p({
+  "class": "foam.nanos.boot.NSpec",
+  "name": "ipGeolocationInfoDAO",
+  "serviceScript":
+  """
+    return new foam.dao.EasyDAO.Builder(x)
+      .setOf(foam.net.ip.IPGeolocationInfo.getOwnClassInfo())
+      .setJournalType(foam.dao.JournalType.SINGLE_JOURNAL)
+      .setJournalName("ipGeolocationInfo")
```
Comment (kgrgreer) [inline src/foam/net/ip/services.jrl:9]: Could/should we .setReadOnly(true)?

File: src/foam/net/ip/IPGeolocationInfo.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
+/**
+ * @license
+ * Copyright 2024 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+
+foam.CLASS({
+  package: 'foam.net.ip',
```
Comment (kgrgreer) [inline src/foam/net/ip/IPGeolocationInfo.js]: 'ip' isn't the most descriptive package name. Maybe 'ipgeo' would be better.
This is server-side only?

Comment (noodlemoodle) [reply src/foam/net/ip/IPGeolocationInfo.js]: will update, and yes

Comment (jlhughes) [reply src/foam/net/ip/IPGeolocationInfo.js]: Then you need to move everything as ip already existed for other ip services/support.

**Review by jlhughes — APPROVED** [review]

**Pull request merged**